Monika After Story/game/zz_hangman.rpy

```diff
@@ -724,10 +675,110 @@ label mas_hangman_game_loop:
+        m 1tsb "...Or maybe you're just biding your time to show me how good you are, hmm?"
+
+        # Make her mention previous episode of giving up on last chance just once.
+        if (
+            mas_seenLabels(["mas_hangman_give_up_last_chance"]) and
+            not mas_seenLabels(["mas_hangman_give_up_last_chance_ref"])
+        ):
+            call mas_hangman_give_up_last_chance_ref
+    else:
+        m 1hua "Wow, you guessed [the_word] correctly!"
+        m "Good job, [player]!"
+
+    if not persistent.ever_won['hangman']:
+        $ persistent.ever_won['hangman'] = True
+        $ mas_gainAffection(2)
+    return
 
-    # RETURN AT END
+label mas_hangman_game_loss(guesses=0, chances=0, last_chance_guesses=0):
+    if guesses > 0:
+        if player_word:
+             m 1eka "[player]..."
+             m 1hksdlb "You couldn't guess your own name?"
+        elif last_chance_guesses / (guesses - (6 - chances)) > 0.5:
+             if not (persistent._mas_pm_cares_about_dokis or persistent._mas_sensitive_mode):
+                 m 1tsa "You were {i}hanging{/i} in there really well{w=0.5}{nw}, "
+                 extend 1hub "[player], ahaha."
+             else:
+                 m 1eua "You were doing really great, [player]!"
+        m 1hua "Better luck next time~"
+    else:
+        m 1ekc "[player]..."
+        m "You didn't guess even a single letter..."
+        m 1eka "I really hope you'll do better next time."
+        if mas_isMoniAff(higher=True):
+            m 1hksdlb "Did I pick a word that was too difficult for you, [mas_get_player_nickname()]?~"
+    return
+
+label mas_hangman_game_give_up(guesses=0, chances=0, last_chance_guesses=0):
+    #hide hmg_hanging_man
+    #show hm_6 zorder 10 as hmg_hanging_man at hangman_hangman
+    m 1lksdlb "[player]..."
+    if guesses == 0:
+        m "I thought you said you wanted to play [store.mas_hangman.game_name]."
+        m 1lksdlc "You didn't even guess a single letter."
+        m "..."
+        m 1ekc "I really enjoy playing with you, you know."
+    elif chances == 1:
+        call mas_hangman_game_give_up_last_chance(guesses, last_chance_guesses)
+    elif chances == 5:
+        m 1ekc "Don't give up so easily."
+        m 3eka "That was just your first wrong letter!"
+        $ number_literal = number_literals[chances]
+        m 1eka "You still had [number_literal] more lives left."
+        m 1hua "I know you can do it!"
+        m 1eka "It would really mean a lot to me if you just tried a bit harder."
+    else:
+        m "You should at least play to the end..."
+        m 1ekc "Giving up so easily is a sign of poor resolve."         
+        if chances >= 2:
+            $ number_literal = number_literals[chances]
+            m "I mean, you'd have to miss [number_literal] more letters to actually lose."
+        else:
+            m "I mean, you'd have to miss one more letter to actually lose."
```

Comment:
I prefer that we keep the number literal instead of converting it into a word. Numbers are more recognizable and universal than their english spelling.

Reply:
That's alright, I'll revert this change.